config/settings.yml

```
@@ -0,0 +1,10 @@
+regex:
+  VALID_EMAIL_REGEX : /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

e thêm ntn nó sẽ hiểu string mà k phải regex
```suggestion
  VALID_EMAIL_REGEX : !ruby/regexp '/\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i'
```